src/preact.d.ts

```diff
@@ -37,6 +37,16 @@ declare namespace preact {
 		new (props?:PropsType):Component<PropsType, StateType>;
 	}
 
+	interface FunctionalComponent<PropsType> {
```

Comment:
In the React typings, the equivalent interface is called `StatelessComponent`. Using the same name would probably be helpful to people migrating from one to the other. Thoughts?